keyboards/dztech/dz65rgbv3/dz65rgbv3.c

```diff
@@ -0,0 +1,136 @@
+    {0, CS33_SW1, CS32_SW1, CS31_SW1},      
+    {0, CS33_SW2, CS32_SW2, CS31_SW2},     
+    {0, CS33_SW3, CS32_SW3, CS31_SW3},     
+    {0, CS33_SW4, CS32_SW4, CS31_SW4}, 
+    {0, CS33_SW5, CS32_SW5, CS31_SW5}, 
+    {0, CS33_SW7, CS32_SW7, CS31_SW7}, 
+    
+    {0, CS9_SW1, CS8_SW1, CS7_SW1},     
+    {0, CS9_SW2, CS8_SW2, CS7_SW2},     
+    {0, CS9_SW3, CS8_SW3, CS7_SW3}, 
+    {0, CS9_SW4, CS8_SW4, CS7_SW4},     
+    {0, CS9_SW5, CS8_SW5, CS7_SW5},     
+    {0, CS9_SW6, CS8_SW6, CS7_SW6},     
+    {0, CS9_SW7, CS8_SW7, CS7_SW7},     
+    {0, CS9_SW8, CS8_SW8, CS7_SW8},    
+    {0, CS36_SW1, CS35_SW1, CS34_SW1},
+    {0, CS36_SW2, CS35_SW2, CS34_SW2},
+    {0, CS36_SW3, CS35_SW3, CS34_SW3},
+    {0, CS36_SW4, CS35_SW4, CS34_SW4},
+    {0, CS36_SW5, CS35_SW5, CS34_SW5},
+    {0, CS36_SW7, CS35_SW7, CS34_SW7},
+    
+    {0, CS3_SW1, CS2_SW1, CS1_SW1},
+    {0, CS3_SW2, CS2_SW2, CS1_SW2},
+    {0, CS3_SW3, CS2_SW3, CS1_SW3},
+    {0, CS3_SW6, CS2_SW6, CS1_SW6},
+    {0, CS39_SW1, CS38_SW1, CS37_SW1},    
+    {0, CS39_SW2, CS38_SW2, CS37_SW2},
+    {0, CS39_SW3, CS38_SW3, CS37_SW3},
+    {0, CS39_SW4, CS38_SW4, CS37_SW4},
+    {0, CS39_SW5, CS38_SW5, CS37_SW5},
+    {0, CS39_SW7, CS38_SW7, CS37_SW7}                   
+                                                                                                     
+};
+led_config_t g_led_config = { {
+    {  0,  1,  2,  3,  4,  5,  6,  7,   8,  9,  10,  11, 12,    13,  14 },
+    {  15, 16, 17, 18, 19, 20, 21, 22,  23, 24, 25,  26, 27,    28,  29 },
+    {  30, 31, 32, 33, 34, 35, 36, 37,  38, 39, 40,  41, NO_LED,42,  43 },
+    {  44, 45, 46, 47, 48, 49, 50, 51,  52, 53, 54,  NO_LED,55, 56,  57 },
+    {  58, 59, 60, NO_LED, NO_LED,  61, NO_LED, NO_LED,  NO_LED, 62, 63,  64,  65,  66,  67 }
+}, {
+    {0,0},{15,0},{30,0},{45,0},{60,0},{75,0},{90,0},{105,0},{120,0},{135,0},{150,0},{165,0},{180,0},{203,0},{224,0},
+	{4,16},{23,16},{38,16},{53,16},{68,16},{83,16},{98,16},{113,16},{128,16},{143,16},{158,16},{173,16},{188,16},{206,16},{224,16},
+	{6,32},{26,32},{41,32},{56,32},{71,32},{86,32},{101,32},{116,32},{131,32},{146,32},{161,32},{176,32},{201,32},{224,32},
+	{9,48},{34,48},{49,48},{64,48},{79,48},{94,48},{109,48},{124,48},{139,48},{154,48},{169,48},{189,48},{210,48},{224,48},
+        {   2,  64 }, {  21,  64 }, {  39,  64 },                                           {  96,  64 },                             { 150,  64 }, { 165,  64 }, { 180,  64 }, { 195,  64 }, { 210,  64 }, { 224,  64 }
+}, {
+    1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+    1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+    1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1, 
+    1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+    1, 1, 1, 4, 1, 1, 1, 1, 1, 1	
+} };
+
+void suspend_power_down_kb(void)
+{
+    rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+}
+
+void suspend_wakeup_init_kb(void)
+{
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
```

Comment:
the `develop` branch should be merged this weekend, and these will no longer be needed.
```suggestion
```